index.bs

```diff
@@ -3022,6 +3023,114 @@ those after the window has reached its new state.
 
 </div>
 
+#### The browser.setDownloadBehavior Command #### {#command-browser-setDownloadBehavior}
+
+A <dfn>download behavior struct</dfn> is a [=struct=] with:
+* [=struct/item=] named <dfn id="download-behavior-struct-allowed" for="download-behavior-struct">allowed</dfn> which is a boolean;
+* [=struct/item=] named <dfn id="download-behavior-struct-destination-folder" for="download-behavior-struct">destinationFolder</dfn> which is a string or null.
+
+A [=remote end=] has a <dfn>download behavior map</dfn> which is a weak map between
+[=user contexts=] or "<code>global</code>" and [=download behavior struct=] or null.
```

Comment:
nit: not sure what's the definition of weak map in spec world, but a "global" string doesn't sound like a typical weak map key?

Reply:
fixed